bootstrap.sh

```diff
@@ -85,6 +85,41 @@ if [[ ${master_settings_missing} == "true" ]]; then
   done
 fi
 
+# Create Elasticsearch snapshot repository
+if ! curl localhost:9200/_snapshot?pretty | grep $BACKUP_REPOSITORY; then
```

Comment:
Tbh, I would rather compare to 
```
[[ "`curl -X GET "localhost:9200/_snapshot/${BACKUP_REPOSITORY}?pretty" | jq .status`" != "404" ]]
```
But `jq` is not present in the image.

Also, if variable is in the middle of string, I would prefer `${BACKUP_REPOSITORY}` to make it more visible, but it doesn't really matter :-) just sayin

Reply:
Or just use `-f` or `--fail` `curl` parameter.